src/slurm_plugin/slurm_resources.py

```diff
@@ -414,7 +427,31 @@ def is_backing_instance_valid(self, log_warn_if_unhealthy=True):
                         self,
                         self.state_string,
                     )
-                return False
+                # Allow a few iterations for the eventual consistency of EC2 data
+                logger.info(
+                    f"ec2_backing_instance_max_count {ec2_backing_instance_max_count} "
+                    f"nodes_without_backing_instance_count_map {nodes_without_backing_instance_count_map}"
```

Comment:
do we want to log this at info level? 
* the ec2_backing_instance_max_count is already logged as clustermgtd config value
* the nodes_without_backing_instance_count_map probably make sense at debug value, but a wording about what it's being logged should be added